src/yb/tserver/heartbeater.cc

```diff
@@ -382,6 +382,19 @@ Status Heartbeater::Thread::TryHeartbeat() {
   req.set_config_index(server_->GetCurrentMasterIndex());
   req.set_cluster_config_version(server_->cluster_config_version());
 
+  // Include the hybrid time of this tablet server in the heartbeat.
+  auto* hybrid_clock = down_cast<server::HybridClock*>(server_->Clock());
```

Comment:
We have tests where `server_->Clock()` is not an instance of `HybridClock`.
This case should be handled.